src/ripple/consensus/Consensus.cpp

```diff
@@ -98,7 +102,7 @@ checkConsensusReached(std::size_t agreeing, std::size_t total, bool count_self)
 
     int currentPercentage = (agreeing * 100) / total;
```

Comment:
Good idea. This was actually a torn refactor--`parms.minCONSENSUS_PCT` is an `int`, but I'm using std::size_t everywhere else.  I'll  fix.